pwa/src/templates/templateParts/actionsForm/ActionFormTemplate.tsx

```diff
@@ -96,13 +101,28 @@ export const ActionFormTemplate: React.FC<ActionFormTemplateProps> = ({ action }
   };
 
   const handleSetFormValues = (): void => {
-    const basicFields: string[] = ["reference", "name", "description", "priority", "async", "isLockable", "isEnabled"];
+    const basicFields: string[] = [
+      "reference",
+      "version",
+      "name",
+      "description",
+      "priority",
+      "async",
+      "isLockable",
+      "isEnabled",
+    ];
     basicFields.forEach((field) => setValue(field, action[field]));
 
     setActionConditionsFieldValue(JSON.stringify(action["conditions"], null, 2));
 
     setValue("class", { label: action.class, value: action.class });
 
+    for (const [key, value] of Object.entries(getUsers?.data)) {
+      if (value?.id === action?.userId) {
+        setValue("userId", { label: setUserIdLabel(value), value: value.id });
+      }
+    }
+

```

Comment:
Also, adding the code here will unfortunately not set the selected user.
The reason is that the function `handleSetFormValues` is only called once after action is done.

To get around this, you will need to add another `React.useEffect` like so:
```
React.useEffect(() => {
  if (!action) return;
  if (!getUser.isSuccess) return;
  // code
  
}, [action, getUser.isSucces]
```
This will unsure that the code will be run when the action is done and getUser is done. When they are both done, you have all the data you need to execute the code successfully